frontend/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -24,6 +24,7 @@ import "./GraphVizChart.scss"
 
 interface Props {
   width: number
+  height: number | undefined
```

Comment:
Unrelated to this line but the UI is a bit jumpy when expanding and resetting the graphviz charts (and probably other charts) but it's easy to see with the second chart in `e2e/scripts/graphviz_chart.py`

Reply:
Yes, I agree about graphviz (no others charts). I guess is because D3 update the chart after component did update.